README.md

```diff
@@ -1,2 +1,19 @@
 # enforce-template-app
-2nd app by RGSoC team $we init
+### 2nd app by RGSoC team $we init
+
+## Statement : 
+* The Bot will require new Issues and PRs to use the respective templates and fill out their
+fields.
+* It will check that the template was actually used instead of deleted by ensuring that
+keywords are present in the body.
+
+## Details of Implementation :
+* The repository on which the bot is to be installed is required to contain an ISSUE_TEMPLATE.md file as well as a PULL_REQUEST_TEMPLATE.md file
+* Upon the opening of an issue or a pull request, the bot is triggered to do the following.
+* It reads the template accordingly (based on whether it is an issue or a pull request that was opened) and then stores the fields present in the template in an array.
```

Comment:
maybe end the "the bot is triggered to do the following" with a ":" and then indent the 3 points below?

Reply:
Done!